typing/out_type.ml

```diff
@@ -1209,6 +1224,11 @@ and tree_of_package mode {pack_path; pack_constraints} =
         (fun (li, ty) -> (String.concat "." li, tree_of_typexp mode ty))
         pack_constraints }
 
+and tree_of_type_path p =
+  let (p', s) = best_type_path p in
+  let p'' = if (s = Id) then p' else p in
+  tree_of_best_type_path p p''
```

Comment:
Moving this function here is strange and potentially confusing for future reader since the function is not used in the recursive group, nor does it have any dependency within the group. I would propose to either let the function in its current position or move it up just after the definition of `tree_of_best_type_path`.

Reply:
Move up, thanks